server/constants/comments.js

```diff
@@ -0,0 +1,8 @@
+
+export const MAXIMUM_AMOUNT_TO_SEND_FORM_COMMENT = 60000; // U$ 600.00 total amount allowed without form as of July 2018
```

Comment:
For the same reason than above, I'd keep this in `collective.settings`.
Maybe a better name for this variable (naming is np-hard!) would be simply `settings.thresholdW9` now that this would be part of the `IRS Bot`. Makes things self explanatory.